torch/nn/modules/module.py

```diff
@@ -1061,6 +1061,9 @@ def to(self, *args, **kwargs):
         .. function:: to(memory_format=torch.channels_last)
            :noindex:
 
+        .. function:: to(memory_format=torch.channels_last_3d)
```

Comment:
Why add this here?
The list here is meant to show all the signatures, not all the possible arguments.

Reply:
I don't know what I'm doing. I thought this would be relevant. Removing it.